JsDocumentation/styles.css

```diff
@@ -0,0 +1,82 @@
+body {
+  display: flex;
+  font-family: "Open Sans", Arial, sans-serif;
+  line-height: 1.5;
+}
+
+#navbar {
+  width: 300px;
+  min-width: 290px;
+  height: 100%;
+  position: fixed;
+  border-right: 2px solid rgba(0, 22, 22, 0.4);
+  top: 0;
+  left: 0;
+}
+
+#main-doc {
+  position: absolute;
+  margin-left: 310px;
+  padding: 0;
+  margin-bottom: 110px;
+  padding: 20px;
+}
+#topic {
+  margin: 10px;
+  font-size: 1.8em;
+  text-align: center;
+}
+
+#navbar ul {
+  color: #4d4e53;
```

Comment:
Always use `color: inherit` whenever possible, instead of specifying the same color in multiple places.